Tribler/Main/vwxGUI/GuiUtility.py

```diff
@@ -112,6 +131,7 @@ def ShowPlayer(self):
 
     @forceWxThread
     def ShowPage(self, page, *args):
+        assert wx.Thread_IsMain()
```

Comment:
@whirm ok, I added them for a failsafe in the future, if someone ever decided to remove them the assert would make it obvious that is a bad idea.